openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -386,6 +384,19 @@ def test_get_account_by_email(self, api_client, user):
         response = self.send_get(client, query_parameters=f'email={self.user.email}')
         self._verify_full_account_response(response)
 
+    def test_get_account_by_user_id(self):
+        """
+        Test that requesting a user id search works.
+        """
+        api_client = "staff_client"
+        user = "staff_user"
+        client = self.login_client(api_client, user)
+        self.create_mock_profile(self.user)
+        set_user_preference(self.user, ACCOUNT_VISIBILITY_PREF_KEY, PRIVATE_VISIBILITY)
+
+        response = self.send_get(client, query_parameters=f'lms_user_id={self.user.id}')
+        self._verify_full_account_response(response)
+
```

Comment:
add test for lms id fetch by a learner account and assert against the response and status code